system/Database/Forge.php

```diff
@@ -271,12 +277,13 @@ public function dropDatabase($db_name)
 	/**
 	 * Add Key
 	 *
-	 * @param    string $key
-	 * @param    bool   $primary
+	 * @param    string|array $key
+	 * @param    bool         $primary
+	 * @param    bool         $unique
 	 *
 	 * @return    Forge
 	 */
-	public function addKey($key, $primary = false)
+	public function addKey($key, $primary = false, $unique = false)
```

Comment:
`$unique` can use `bool` type hint I think .: `bool $unique = false`

Reply:
Yes, and the `$primary` too because it is compared with `===`.